examples/input/example.gts

```diff
@@ -1,12 +1,14 @@
 import type { TemplateOnlyComponent } from '@ember/component/template-only';
 
 export interface Signature {
-  Element: HTMLElement,
-  Args: {
-
-
-  }
-  Yields: []
+  Element: HTMLElement;
```

Comment:
I think we can revert the changes in this file.

Reply:
fixed